src/components/BookItem.jsx

```diff
@@ -2,23 +2,43 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { useDispatch } from 'react-redux';
 import { deleteBook } from '../redux/books/booksSlice';
+import { CircularProgressBar } from './Icons';
 
 const BookItem = ({ book }) => {
+  const randomPercentage = Math.floor(Math.random() * 101);
   const dispatch = useDispatch();
   return (
-    <>
-      <span>
-        <strong>Title:</strong>
-        {book.title}
-        ,
-        <strong>Author:</strong>
-        {book.author}
-        ,
-        <strong>Category:</strong>
-        {book.category}
-      </span>
-      <button type="button" onClick={() => dispatch(deleteBook(book))}>Delete</button>
-    </>
+    <div className="single-book-container">
+      <div className="single-book">
+        <div className="book-info">
+          <span className="book-category">{book.category}</span>
+          <span className="book-title">{book.title}</span>
+          <span className="book-author">{book.author}</span>
+        </div>
+        <div className="btn-container">
+          <button type="button">Comments</button>
+          <span>|</span>
+          <button type="button" onClick={() => dispatch(deleteBook(book))}>Remove</button>
+          <span>|</span>
+          <button type="button">Edit</button>
+        </div>
+      </div>
+      <div className="mid-section">
+        <CircularProgressBar percentage={randomPercentage} />
+        <div>
+          <span className="percentage-text">
+            {randomPercentage}
+            %
+          </span>
```

Comment:
- [ ] Well done! The color for this percentage text matches the design perfectly. However, upon closer inspection, it's evident that the font size doesn't align with the design's specifications. To maintain design consistency, I kindly suggest updating your code to match the design's intended font size for the percentage text. This attention to detail will help ensure that all aspects of your project adhere to the design's aesthetic.

Keep up the great work, and thank you for your commitment to refining your project! 🌟🔤